src/Components/Dashboard/Dashboard.tsx

```diff
@@ -1,27 +1,59 @@
 import React, { useEffect, useState } from 'react';
-import { gql, useQuery } from "@apollo/client";
-import { GET_SINGLE_USER } from '../../Queries';
+import { useQuery, useLazyQuery } from "@apollo/client";
+import { GET_SINGLE_USER, DELETE_STORY } from '../../Queries';
 import './Dashboard.scss';
 import UserInfo from '../UserInfo/UserInfo';
 import SingleStory from '../SingleStory/SingleStory'
 import '../Loader/Loader.tsx';
 import Loader from '../Loader/Loader';
+import { useMutation } from '@apollo/client';
 
 interface IStory {
   id: string,
   title: string,
   word: string,
-  createdAt: string
+  createdAt: string,
 }
 
+
 const Dashboard: React.FC = () => {
-  const { loading, error, data } = useQuery(GET_SINGLE_USER, {
+  // const [ userStories, setUserStories ] = useState([])
+  const  { loading, error, data }  = useQuery(GET_SINGLE_USER, {
     fetchPolicy: "no-cache",
     variables: {id: 1},
-  })
+  });
+
+
+  const [ deleteStory, {
+    data: deleteData,
+    loading: deleteLoading,
+    error: deleteError
+  }] = useMutation(DELETE_STORY)
+
+  // const getUserInfo = async () => {
+  //   console.log('async data', data)
+  //   const result = await data;
+  //   setUserStories(result.fetchUser.stories)
+  // }
+  // useEffect( (): void => {
+  //   console.log('data', data)
+  //   getUserInfo()
+  // }, [data])
+
   if (loading) return <Loader/>
   if (error) return <p>We're sorry, there's been an error! Please try again.</p>
 
+  const deleteStoryFromDom = (id: string) => {
+    let filteredStories = data.fetchUser.stories.filter((story: IStory) => {
+      return story.id !== id
+    })
+    let idNumber: number = parseInt(id)
+    deleteStory( {
+      variables: {id: idNumber}
+    })
+    data.fetchUser.stories = filteredStories
```

Comment:
Way to manually delete the story from state so that we don't have to make another request to the server.

Reply:
Nice!!!